src/praisonai-code/praisonai_code/cli/commands/plugins.py

```
@@ -272,6 +275,34 @@ def plugins_doctor():
+    Forces rediscovery of single-file and entry-point plugins, then rewires
+    enabled plugins into the runtime hook registry — so newly added plugins
+    take effect in the current process instead of on the next run.
+
+    Examples:
+        praisonai plugins reload
+    """
+    try:
+        from praisonaiagents.plugins.manager import get_plugin_manager
+    except ImportError:
+        typer.echo("Error: praisonaiagents package not found.", err=True)
+        raise typer.Exit(1)
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)